Makefile

```diff
@@ -1,4 +1,4 @@
-TARGETS=go1.10 go1.11 go1.12 go1.13 go1.14 go1.15 go1.16
+TARGETS=go1.15 go1.16
```

Comment:
@kvch shall we keep go1.13 and go1.14? So we can test the functionbeat build with the version used by GCP?

Reply:
IIIUC, those docker images are available anyway since they were built already but since they are not patched anymore we don't need to keep their configuration

Reply:
Sounds good.